doc/fping.pod

```diff
@@ -155,6 +155,12 @@ to any target (default is 10, minimum is 1).
 
 Set the interface (requires SO_BINDTODEVICE support).
 
+=item B<--icmp-timestamp>
+
+Send ICMP timestamp requests (ICMP type 13).
+If B<fping> not recive ICMP timestamp reply, "(Timestamp unknown)" is returned.
```

Comment:
Please insert the word "does" before "not".

Reply:
Superfluous due to the omission of `(Timestamp unknown)`